CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 10.4
```

Comment:
Didn't it mean that updating the CapiModelsVersion to version 10.3 triggered the content-api-scala-client to move to 10.4 ?
(Or did I get something wrong ?)